src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -73,6 +75,22 @@ def validate_create_parameters(namespace):
         raise CLIError('--dns-prefix has no value')
 
 
+def validate_ip_ranges(namespace):
+    if not namespace.api_server_authorized_ip_ranges:
+        return
+
+    allow_all_traffic = "0.0.0.0/32"
```

Comment:
"dis"allow_all_traffic

Reply:
Good catch - set to restrict_traffic_to_agentnodes